src/utils/wallet.ts

```diff
@@ -475,13 +499,26 @@ const wallet = {
       };
     }
 
-    return {
+    const accessData: IWalletAccessData = {
       walletType,
       multisigData,
       mainKey: encryptedMainKey,
       xpubkey: xpriv.xpubkey,
       walletFlags: 0,
     };
+
+    if (acctXpriv !== null) {
+      // Account path key will only be available if the provided key is a root key
+      const encryptedAcctPathKey = encryptData(acctXpriv.xprivkey, pin);
+      accessData.acctPathKey = encryptedAcctPathKey;
+    }
+
+    if (!!(seed && password)) {
+      const encryptedWords = encryptData(seed, password);
+      accessData.words = encryptedWords;
+    }
+
```

Comment:
In order to use this method instead of `generateAccessDataFromSeed`, we would need to also add `authKey`. Does it make sense to add it?

Reply:
Makes sense, added authKey when starting with xpriv as well.
But since the path to the authKey is special, we can only do so when starting from the root xprivkey